Felienne/hedy
# Bug Fix: Calculator game at level 14

**Description**
Changed indentation at YAML file to make calculator game at level 14 work.


**Fixes #2420 **


**How to test**
Go to level 14, go to the calculator game. Copy paste the example code and run it. It should be running without a problem.
